src/screens/home/components/mentee-home/mentee-home.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
Not sure about separating components into mentee and non-mentee inside the home screen directory. Potentially there could be a lot more changes between those screen and maybe we have to split them on the screen level

Reply:
Done (moved to screen level)